### DISCUSSION TITLE
使用flusher_http发送请求的时候，并没有成功

### DISCUSSION THREAD

**Discussion by hasylu** [alibaba/ilogtail#953] [Help] [answered]
配置文件是参考官方的，大致如下（其实是发到企业微信）：
enable: true
inputs:
  - Type: file_log
    LogPath: /home/test-log/
    FilePattern: "*.log"
flushers:
  - Type: flusher_http
    RemoteURL: "http://inner.nginx.com/test"
    Convert:
      Protocol: custom_single
      Encoding: json

为了做一个测试，在内网nginx写了个测试接口，发送写日志的时候，并没有请求到这个接口，特地加上flusher_stdout，发现stdout是正常的，但是http还是没有请求
flushers:
  - Type: flusher_http
    RemoteURL: "http://inner.nginx.com/test"
    Convert:
      Protocol: custom_single
      Encoding: json
  - Type: flusher_stdout
    OnlyStdout: true

ilogtail的日志文件没有特别的输出

**Comment by messixukejia** [comment, marked as answer]
可以看下插件日志：https://ilogtail.gitbook.io/ilogtail-docs/configuration/logging
